sashimi-webapp/src/logic/documentPackager/conditionalProcessor.js

```diff
@@ -235,9 +235,17 @@ function hideShowBlock(state, startLine, endLine, silent) {
   return true;
 }
 
-export default function hideShowPlugin(md) {
-  md.inline.ruler.after('emphasis', 'hideShowInline', hideShowInline);
-  md.block.ruler.after('table', 'hideShowBlock', hideShowBlock);
-}
+export default {
+  hideShowPlugin: function hideShowPlugin(md) {
+    md.inline.ruler.after('emphasis', 'hideShowInline', hideShowInline);
+    md.block.ruler.after('table', 'hideShowBlock', hideShowBlock);
+  },
+  setFileName: function setFileName(name) {
+    this.fileName = name;
```

Comment:
`hideShowInline` and `hideShowBlock` won't have access to this.fileName.

Not sure if there is a better way to do this, but you can consider replacing `this.fileName` to `fileName` (using it like a private variable in the `conditionalProcessor.js`.